apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.sql

```diff
@@ -87,3 +87,22 @@ WINDOW
   )
 ORDER BY
   qo.row_order;
+
+-- BLOCK mark_variant_broken
+UPDATE variants AS v
+SET
+  broken_at = CURRENT_TIMESTAMP,
+  broken_by = $authn_user_id
+WHERE
+  id IN (
+    SELECT
+      v.id
+    FROM
+      variants v
+      JOIN instance_questions iq ON v.instance_question_id = iq.id
+    WHERE
+      v.open = true
+      AND v.broken_at IS NULL
+      AND iq.id = $instance_question_id
+      AND iq.assessment_instance_id = $assessment_instance_id
+  );
```

Comment:
```suggestion
-- BLOCK mark_variant_broken
UPDATE variants AS v
SET
  broken_at = CURRENT_TIMESTAMP,
  broken_by = $authn_user_id
FROM
  instance_question iq
WHERE
  v.instance_question_id = iq.id
  AND v.open = true
  AND v.broken_at IS NULL
  AND iq.id = $instance_question_id
  AND iq.assessment_instance_id = $assessment_instance_id;
```

Reply:
I edited your comment to fix `instance_question` to `instance_questions`. Looks good; this was also copied blindly from the original implementation.